src/ui/Windows/Properties.cpp

```diff
@@ -46,9 +48,18 @@ BOOL CProperties::OnInitDialog()
 {
   CPWDialog::OnInitDialog();
 
-  CString ngroups;
+  CString ngroups, cs_numGroups;
+#if 0
+  // Use groups based on the entries
   ngroups.Format(IDS_NUMGROUPS_E,
      m_pdbp->numgroups.c_str(), m_pdbp->numemptygroups.c_str());
+#endif
+  // Use groups based on the groups that are displayed in Tree view
+  // cf. Windows Explorer
+  std::vector<std::wstring> vGroups;
+  GetMainDlg()->GetAllGroups(vGroups);
```

Comment:
Perhaps this should be the semantics of m_pdbp->numgroups? Is m_pdbp->numgroups used anywhere else that might be broken by changing this?
In any case, wx's property display code should be similarly changed, otherwise the two versions will show different counts for the same DB...